tools/pontoon/users.md

```diff
@@ -0,0 +1,65 @@
+# Managing users and profile settings
+
+* [Creating an account](#creating-an-account).
+* [Profile settings](#profile-settings).
+* [User roles](#user-roles).
+* [Managing permissions](#managing-permissions).
+
+## Creating an account
+
+Pontoon currently uses Firefox Accounts to manage users, so you will need to [create an account](https://accounts.firefox.com/signup) in order to log in.
+
+To create an account on [Pontoon](https://pontoon.mozilla.org/), simply click on the menu icon in the top right corner and select *Sign In*.
+
+![Pontoon login](/assets/images/pontoon/users/menu_login.png)
+
+## Profile settings
+
+[User settings](https://pontoon.mozilla.org/settings/) can be accessed by clicking the user avatar in the top right corner, and selecting *Settings*.
+
+![Pontoon login](/assets/images/pontoon/users/menu_settings.png)
+
+Profile picture can be set on [Gravatar](http://gravatar.com/), while the displayed name can be edited simply by clicking on it.
+
+![Pontoon login](/assets/images/pontoon/users/profile_page.png)
+
+Both *Quality checks* and *Make suggestions* are checkboxes. In the picture above *Quality checks* is disabled while *Make suggestions* is enabled.
+* *Quality checks* (check for number of sentences, placeholders, HTML tags, punctuation, etc.) are performed when saving a translation.
+* *Make suggestions* allows a translator or a manager to submit suggestions instead of translations.
+
+Both settings can also be changed directly from the gear menu (highlighted in orange) when translating strings. Profile page is also accessible from this menu by selecting *Change all settings*.
+
+![Pontoon login](/assets/images/pontoon/users/translation_gear.png)
+
+*Preferred locales* can be used to display a specific group of locales before others in the *LOCALES* tab when translating strings (highlighted in red). In this case, the user chose to display fr and es-ES before other translations, since they’re both available the tab displays 2+72, meaning 2 preferred locales plus all others.
```

Comment:
Maybe put *fr* and *es-ES* in italic?

For easier readability, maybe split the last sentence into two? **translations, since** -> **translations. Since**